src/screens/ProfileScreen/TaskScreens/NoteWorthy.tsx

```diff
@@ -103,19 +75,69 @@ const Note = () => {
           ))}
         </View>
       ) : (
-        <View
-          style={{
-            justifyContent: 'flex-start',
-            alignItems: 'center',
-          }}
-        >
-          <Text style={{ fontSize: 13, color: 'black' }}>
-            No noteworthy task yet!
-          </Text>
+        <View style={styles.noTaskContainer}>
+          <Text style={styles.noTaskText}>No noteworthy task yet!</Text>
         </View>
       )}
     </ScrollView>
   );
 };
 
 export default Note;
+
+const styles = StyleSheet.create({
```

Comment:
We can put this into a separate file.

Reply:
We can only put it in separate file, if styles are in very large no of lines.

Reply:
There is someone who is already working on similar issue where the stylesheet is being removed from component file. So, just for consistency, I asked it.

Reply:
Yes but only for the files only whose stylesheet is longer causing component unreadable.